packages/spotlight/Spottable/Spottable.js

```diff
@@ -316,14 +316,16 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 		}
 
 		handleFocus = (ev) => {
-			if (ev.currentTarget === ev.target) {
-				this.isFocused = true;
-			}
+			if (!this.props.disabled) {
```

Comment:
Let's update this to be an early bail out to minimize the diff and keep the code as readable as possible.

```js
handleFocus = (ev) => {
  if (this.props.disabled) return;
  ...
}
```